crates/turbopack-ecmascript/src/lib.rs

```diff
@@ -95,14 +95,21 @@ fn modifier() -> StringVc {
 }
 
 #[turbo_tasks::value]
-#[derive(Clone, Copy)]
 pub struct EcmascriptModuleAsset {
     pub source: AssetVc,
     pub context: AssetContextVc,
     pub ty: EcmascriptModuleAssetType,
     pub transforms: EcmascriptInputTransformsVc,
     pub compile_time_info: CompileTimeInfoVc,
     pub inner_assets: Option<InnerAssetsVc>,
+    #[turbo_tasks(debug_ignore)]
+    pub last_successful_analyse: turbo_tasks::State<
+        Option<(
+            AnalyzeEcmascriptModuleResultVc,
+            AssetReferencesReadRef,
+            EcmascriptExportsReadRef,
```

Comment:
Put these three into a struct called `MemoizedSuccessfulAnalysis` or something like that.